rendering/markup_content_test.go

```diff
@@ -0,0 +1,33 @@
+package rendering
+
+import "testing"
+import "github.com/stretchr/testify/assert"
+import "github.com/stretchr/testify/require"
+
+func TestGetDefaultMarkupFromNil(t *testing.T) {
+	// when
+	result := NilSafeGetMarkup(nil)
+	// then
+	require.NotNil(t, result)
+	assert.Equal(t, SystemMarkupDefault, result)
+}
+
+func TestGetMarkupFromValue(t *testing.T) {
+	// given
+	markup := SystemMarkupMarkdown
+	// when
+	result := NilSafeGetMarkup(&markup)
+	// then
+	require.NotNil(t, result)
+	assert.Equal(t, markup, result)
+}
+
+func TestGetMarkupFromEmptyValue(t *testing.T) {
+	// given
+	markup := ""
+	// when
+	result := NilSafeGetMarkup(&markup)
+	// then
+	require.NotNil(t, result)
```

Comment:
I'm not sure you need this nil check because the `assert.Equal` below just compares values and it is safe to compare nil values as long as you don't dereference a nil value. Right?